packages/theme/jest.config.ts

```diff
@@ -0,0 +1,10 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
```

Comment:
I'm aware of having `jest.base.config` file at the root directory, although importing/requiring it from the `jest.config.ts` file at the root of the theme package results with an error (It's because `"rootDir": "./",` in the `tsconfig.json`, and `jest.base.config` is outside of the root scope). Can we fix it somehow, to make use of the default config (as done in `/spree-api/jest.config.js` for instance)?